providers/aws/services/accessanalyzer/accessanalyzer_enabled_without_findings/accessanalyzer_enabled_without_findings_test.py

```diff
@@ -0,0 +1,164 @@
+from unittest import mock
+
+from providers.aws.services.accessanalyzer.accessanalyzer_service import Analyzer
+
+
+class Test_accessanalyzer_enabled_without_findings:
+    def test_no_analyzers(self):
+        accessanalyzer_client = mock.MagicMock
+        accessanalyzer_client.analyzers = []
+        with mock.patch(
+            "providers.aws.services.accessanalyzer.accessanalyzer_service.AccessAnalyzer",
+            new=accessanalyzer_client,
+        ):
+            # Test Check
+            from providers.aws.services.accessanalyzer.accessanalyzer_enabled_without_findings.accessanalyzer_enabled_without_findings import (
+                accessanalyzer_enabled_without_findings,
+            )
+
+            check = accessanalyzer_enabled_without_findings()
+            result = check.execute()
+
+            assert len(result) == 0
+
+    def test_one_analyzer_not_available(self):
+        # Include analyzers to check
+        accessanalyzer_client = mock.MagicMock
+        accessanalyzer_client.analyzers = [
+            Analyzer(
+                "",
+                "Test Analyzer",
+                "NOT_AVAILABLE",
+                "",
+                "",
+                "",
+                "eu-west-1",
+            )
+        ]
+        with mock.patch(
+            "providers.aws.services.accessanalyzer.accessanalyzer_service.AccessAnalyzer",
+            accessanalyzer_client,
+        ):
+            from providers.aws.services.accessanalyzer.accessanalyzer_enabled_without_findings.accessanalyzer_enabled_without_findings import (
+                accessanalyzer_enabled_without_findings,
+            )
+
+            check = accessanalyzer_enabled_without_findings()
+            result = check.execute()
+
+            assert len(result) == 1
```

Comment:
Again, test the result of the execution of the check

Reply:
Done, thanks!